mlflow/deployments/cli.py

```diff
@@ -258,3 +258,24 @@ def predict(target, name, input_path, output_path):
             predictions_to_json(result, fp)
     else:
         predictions_to_json(result, sys.stdout)
+
+
+@commands.command("explain")
+@deployment_name
+@target_details
+@parse_input
+@parse_output
+def explain(target, name, input_path, output_path):
+    """
+    predict the features importance for the deployed model for the given input(s)
```

Comment:
Could we enhance this docstring to make it clearer what the API does? Here's a stab at a docstring, let me know if I'm understanding the purpose of this API correctly and thanks in advance!:

```suggestion
    """
    Generate explanations of model predictions on the specified input for the deployed model for the given input(s). Explanation output formats vary by deployment target, and can include details like feature importance for understanding/debugging predictions. Run `mlflow deployments help` or consult the documentation for your plugin for details on explanation format. For information about the input data formats accepted by this function, see the following documentation: https://www.mlflow.org/docs/latest/models.html#built-in-deployment-tools
    """    
```

@shrinath-suresh it'd be good to check - given this explanation, could a new user successfully understand how to generate/interpret model explanations from a model deployed to torchserve?